src/pathways/escher.service.ts

```diff
@@ -1,100 +1,135 @@
 import * as escher from 'escher-vis'
 import * as d3 from 'd3';
+import * as _ from 'lodash';
+import * as fp from 'lodash/fp';
 import 'jquery';
 
+/**
+ * Plan: the current structure is the following:
+ * pathway: {
+ *   reactions <= sorted reactions
+ *   primary_nodes <= the primary nodes belonging to each reaction
+ *   model <= model that ALSO contains reactions, but in the different order
+ * }
+ * I think reactions and primary_nodes should be merged together
+*/
+
 export class EscherService {
-	direction: number;
-	startCoordinates: any;
-	options: any;
-	metabolitesNames: any[];
-	reactions: any[];
-	self: EscherService;
+    direction: number;
+    options: any;
```

Comment:
Is there a type defined for these options? If not, I think it would be wise to define one to avoid using `any`.